src/Commands/MakeRoutesCommand.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Regnerisch\LaravelBeyond\Commands;
+
+use Illuminate\Console\Command;
+
+class MakeRoutesCommand extends Command
+{
+    protected $signature = 'beyond:make:route {application}';
+
+    protected $description = 'Make a new file for routes';
+
+    public function handle()
+    {
+        $application = $this->argument('application');
+
+        beyond_copy_stub(
+            'routes.stub',
+            app_path() . "/../src/App/{$application}/routes.php",
+            [
+                '{{ application }}' => $application,
+            ]
+        );
```

Comment:
We could output the Code which needs to be added by the User in `RouteServiceProvider` or at least give him the information to add the route file manually there.